frontend/fingertips-frontend/mock/data/healthdata.ts

```diff
@@ -1960,6 +1965,7 @@ export const mockHealthData: Record<string, HealthDataForArea[]> = {
           ageBand: '5-9',
           sex: 'Persons',
           trend: HealthDataPointTrendEnum.NotYetCalculated,
+          isAggregate: true,
```

Comment:
Again, think this should be false:
```suggestion
          isAggregate: false,
```